packages/components/src/Notification/Notification.component.js

```diff
@@ -67,27 +71,29 @@ export function TimerBar({ type, autoLeaveError }) {
 }
 
 export function Notification({ notification, leaveFn, ...props }) {
-	const notificationClasses = {
-		[theme['tc-notification']]: true,
-		'tc-notification': true,
-
+	const isError = notification.type === 'error';
+	const classes = classNames(theme['tc-notification'], 'tc-notification', {
 		[theme['tc-notification-info']]: !notification.type || notification.type === 'info',
 		'tc-notification-info': !notification.type || notification.type === 'info',
 
 		[theme['tc-notification-warning']]: notification.type === 'warning',
 		'tc-notification-warning': notification.type === 'warning',
 
-		[theme['tc-notification-error']]: notification.type === 'error',
-		'tc-notification-error': notification.type === 'error',
-	};
-	const classes = classNames(notificationClasses);
+		[theme['tc-notification-error']]: isError,
+		'tc-notification-error': isError,
+	});
+	const enterAction = event => props.onMouseEnter(event, notification);
+	const leaveAction = event => props.onMouseOut(event, notification);
+
 	return (
-		<div // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
I think we actually need this eslint-disable?

Reply:
Same here, I revert